share/completions/cmd.fish

```diff
@@ -0,0 +1,50 @@
+function __cmd_generate_args --description 'Function to generate args'
```

Comment:
This function does not generate the arguments, but actually completes the command-line; it would be better to call it (and similar functions) `__fish_complete_cmd` or similar.